lib/jekyll-import/importers/mt.rb

```diff
@@ -58,15 +65,24 @@ def self.specify_options(c)
       def self.process(options)
         options  = default_options.merge(options)
 
+        engine   = options.fetch('engine')
         dbname   = options.fetch('dbname')
         user     = options.fetch('user')
         pass     = options.fetch('password', "")
         host     = options.fetch('host', "localhost")
+        port     = options.fetch('port')
```

Comment:
`.fetch()` requires that a `port` is specified. to give it a default, do `options.fetch('port', 5432)` or, if it doesn't matter, you can just do `options['port']`.